pkg/utils/clients/factories.go

```diff
@@ -68,7 +67,7 @@ func NewHTTPClient(ctx context.Context, maxConnsPerHost MaxConnsPerHost) *nethtt
 				MaxConnsPerHost:     int(maxConnsPerHost),
 				IdleConnTimeout:     30 * time.Second,
 			},
-			Propagation: &tracecontext.HTTPFormat{},
+			Propagation: tracecontextb3.TraceContextEgress,
```

Comment:
This shouldn't be necessary as the client is already configured to use tracecontext.

Reply:
I think you're right, this isn't really needed as this function returns an `http.Client`, which is egress only. However, I would like to change it to be consistent with all other places in the repo. The actual difference in runtime will be miniscule.

Reply:
Can we just use the regular tracecontext format everywhere we don't need special behavior? That would maintain consistency and I feel that it is somewhat confusing to configure the client with a format that has special unneeded ingress behavior. I reviewed the tracecontextb3 PR before the formats were split but it should actually be unnecessary to have separate egress and ingress formats since the format is only every used with a handler (ingress) or a client transport (egress).

Reply:
> Can we just use the regular tracecontext format everywhere we don't need special behavior? That would maintain consistency

Consistency with what? For the rest of this comment I will assume it is consistency within this repo or knative more generally. As opposed to consistency with the wider world of software using OpenCensus.

I disagree that it is simpler or more consistent to use `&tracecontext.HTTPFormat{}` for ingress and `tracecontextb3.TraceContextEgress` for egress, rather than always using `tracecontextb3.TraceContextEgress` for both. There is some runtime overhead, but it is going to be effectively imperceptible (one stack frame and an iteration of a size one slice per HTTP request, AFAICT). 

If we always use the same format, then developers don't have to understand when to use each. They just always use `tracecontextb3.TraceContextEgress`, which gives the desired behavior in both circumstances.